apps/src/labs/header/ProjectWrappedShareDialog.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Wrapper around ShareDialog that plumbs in the necessary props for a project
+ * using the Project Manager and the labs system.
+ */
+import ShareDialog from '@cdo/apps/code-studio/components/ShareDialog';
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {LabState} from '../labRedux';
+import {isSignedIn as getIsSignedIn} from '@cdo/apps/templates/currentUserRedux';
+import popupWindow from '@cdo/apps/code-studio/popup-window';
+
+const ProjectWrappedShareDialog: React.FunctionComponent<
+  ProjectWrappedShareDialogProps
+> = ({shareUrl}) => {
+  const isProjectLevel = useSelector(
+    (state: {lab: LabState}) => state.lab.isProjectLevel
+  );
+  const appType = useSelector(
+    (state: {lab: LabState}) => state.lab.channel?.projectType
+  );
+  const channelId = useSelector(
+    (state: {lab: LabState}) => state.lab.channel?.id
+  );
+  const isSignedIn: boolean = useSelector(
+    (state: {
+      currentUser: {signInState: 'Unknown' | 'SignedIn' | 'SignedOut'};
+    }) => getIsSignedIn(state.currentUser)
+  );
+  const is13Plus = useSelector(
+    (state: {currentUser: {under13: boolean}}) => !state.currentUser.under13
+  );
+
+  // We don't current support dance party projects in labs.
```

Comment:
ultra nit: `currently`

Also, what does "labs" refer to here?

Reply:
Starting a thread on naming [in slack](https://codedotorg.slack.com/archives/C044AGTKW3D/p1688154551001169). Naming things in this pr was a struggle for me.